src/ancrec.h

```diff
@@ -48,6 +48,18 @@ typedef basAnc* BP;	// basAnc pointer -- formerly used to localize NEARness
 	         re field-interdependent error messages and defaults.
 	      3. may be tested by cuprobe.:probe() re input time probes. */
 
+#if 0
+// unsuccessful experiment re general fn type check, 8-2023
+template <typename T> struct FLDNUM
+{	FLDNUM(T _fn)
+	{
+		static_assert(std::is_same<int, T>::value);
+		fn = _fn;
+	}
+	int fn;
+};
+#endif
+
```

Comment:
Doest this need to be kept? If it has promise, maybe it can be moved to an issue instead?

Reply:
Agree, I will delete.